src/java.base/share/classes/jdk/internal/classfile/impl/DirectCodeBuilder.java

```diff
@@ -1189,6 +1189,67 @@ public CodeBuilder iinc(int slot, int val) {
         return this;
     }
 
+
+    @Override
+    public CodeBuilder iload(int slot) {
+        writeLocalVar(BytecodeHelpers.iload(slot), slot);
+        return this;
+    }
+
+    @Override
+    public CodeBuilder lload(int slot) {
```

Comment:
Uh, I mean each of the method is inserted to our existing override list by alphabetical order instead of inserting everything to the same place at once